src/TinyUPnP.cpp

```diff
@@ -6,27 +6,29 @@
 #include "Arduino.h"
 #include "TinyUPnP.h"
 
+#ifdef UPNP_DEBUG
```

Comment:
Thank you for that :)
Can you please use the original methods?
UPNP_DEBUG --> debugPrint
UPNP_DEBUGln --> debugPrintln